frontend/components/Domain/Recipe/RecipePage/RecipePage.vue

```diff
@@ -104,6 +104,18 @@
         :recipe="recipe"
         :scale="scale"
       />
+      <div class="px-2 px-md-4 pb-4 ">
+        <v-card class="elevation-2">
+          <v-card-title>{{ $t('recipe.not-linked-ingredients') }}</v-card-title>
+            <RecipeIngredients
+              :value="notLinkedIngredients"
+              :scale="scale"
+              :disable-amount="recipe.settings.disableAmount"
+              :is-cook-mode="isCookMode">
+
+            </RecipeIngredients>
+        </v-card>
+      </div>
```

Comment:
I think it would actually make sense to have this not in a card. 
When it's not in a card it is clearly different from items above which are Recipe Steps which this is not. 

This also solves the problem of the card behaving differently (not being collapsable)

Reply:
I Added a divider and made the card flat, making it clearer that its not a instruction